package.json

```diff
@@ -26,10 +26,13 @@
     "react": "^15.5.4",
     "react-bootstrap": "^0.31.0",
     "react-color": "^2.12.1",
+    "react-container-dimensions": "^1.3.3",
```

Comment:
You haven't used this package anywhere.

Reply:
@akshatnitd , this package was required for react-facebook-login , other wise it was showing unmet dependencies